spec/fizzbuzz_spec.rb

```diff
@@ -0,0 +1,7 @@
+require_relative '../fizzbuzz'
+
+describe Fizzbuzz do
+  it 'tests the method fizzbuzz' do
+    expect(subject.fizzbuzz(15)).to eq('fizzbuzz')
+  end
+end
```

Comment:
- [ ] kindly add test cases for when

 ### N is divisible by 3 
 ### N is divisible by 3 and 5
 ### N is not divisible by 3 or 5